packages/boot/tools/liquidation.ts

```diff
@@ -63,52 +63,13 @@ export const likePayouts = ({ Bid, Collateral }) => ({
   },
 });
 
-export const makeLiquidationTestContext = async t => {
-  console.time('DefaultTestContext');
-  const swingsetTestKit = await makeSwingsetTestKit(t.log, 'bundles/vaults', {
-    configSpecifier: '@agoric/vm-config/decentral-main-vaults-config.json',
-  });
-
-  const { runUtils, storage } = swingsetTestKit;
-  console.timeLog('DefaultTestContext', 'swingsetTestKit');
-  const { EV } = runUtils;
-
-  // Wait for ATOM to make it into agoricNames
-  await EV.vat('bootstrap').consumeItem('vaultFactoryKit');
-  console.timeLog('DefaultTestContext', 'vaultFactoryKit');
-
-  // has to be late enough for agoricNames data to have been published
-  const agoricNamesRemotes: AgoricNamesRemotes =
-    makeAgoricNamesRemotesFromFakeStorage(swingsetTestKit.storage);
-  const refreshAgoricNamesRemotes = () => {
-    Object.assign(
-      agoricNamesRemotes,
-      makeAgoricNamesRemotesFromFakeStorage(swingsetTestKit.storage),
-    );
-  };
-  agoricNamesRemotes.brand.ATOM || Fail`ATOM missing from agoricNames`;
-  console.timeLog('DefaultTestContext', 'agoricNamesRemotes');
-
-  const walletFactoryDriver = await makeWalletFactoryDriver(
-    runUtils,
-    storage,
-    agoricNamesRemotes,
-  );
-  console.timeLog('DefaultTestContext', 'walletFactoryDriver');
-
-  const governanceDriver = await makeGovernanceDriver(
-    swingsetTestKit,
-    agoricNamesRemotes,
-    walletFactoryDriver,
-    // TODO read from the config file
-    [
-      'agoric1gx9uu7y6c90rqruhesae2t7c2vlw4uyyxlqxrx',
-      'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
-      'agoric14543m33dr28x7qhwc558hzlj9szwhzwzpcmw6a',
-    ],
-  );
-  console.timeLog('DefaultTestContext', 'governanceDriver');
-
+export const makeLiquidationTestKit = async ({
+  swingsetTestKit,
+  agoricNamesRemotes,
+  walletFactoryDriver,
+  governanceDriver,
+  like,
```

Comment:
please maintain the `t.` since it's using the Ava API (even if benchmarkerator doesn't have full Ava).

```suggestion
  t,
```
It's type can be `t: Pick<ExecutionContext, 'like'>`

Reply:
I think I need some additional TS syntax support on this one.  `tsc` complains:
`error TS2693: 'Pick' only refers to a type, but is being used as a value here.`
It looks to _me_ like a type annotation, but apparently I'm not holding it right.

Reply:
I couldn't repro that but I did run into another kink, the API has a `.skip` property: https://github.com/avajs/ava/blob/af5684dff58a79da862fa57fad0e946985f730a2/types/assertions.d.cts#L186-L194

For now I just pushed a commit that suppresses the problem. (Along with some annotations for drivers)

Reply:
Weirdly, `tsc` is happy if I leave the type of the partial implementation of `t` undeclared.

Reply:
it's only silent because there's no type annotation for `t` in the called function. Your force push removed https://github.com/Agoric/agoric-sdk/commit/a0dce2747cacc50dd76e0447ddb0e6cfa1542d85 and https://github.com/Agoric/agoric-sdk/commit/9b5062f8526e7d4b42bdaff8606fa54e25179394

I'm pushing another fix on top of your change.